src/kubelet/group.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 
 	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
```

Comment:
This should be imported as `corev1`.

Reply:
I could change it but it is imported as `v1` across the whole codebase :(
![image](https://user-images.githubusercontent.com/969721/130605815-0249a6b9-beb8-4618-b584-3c4966005095.png)

Reply:
It's consistent, so not ideal, but good enough :+1: